_topic_map.yml

```diff
@@ -2960,10 +2954,20 @@ Topics:
     File: cluster-logging-serverless
   - Name: Splitting traffic between revisions
     File: splitting-traffic-between-revisions-knative
+  # Tracing
+  - Name: Tracing requests
```

Comment:
Should this be _Tracing requests using Jaeger_ to match the title of the linked assembly?